Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -796,22 +798,45 @@ extension BrowserViewController: WKNavigationDelegate {
     }
 
     func webView(webView: WKWebView, didFinishNavigation navigation: WKNavigation!) {
+        let tab: Browser! = tabManager.getTab(webView)
+        if tab == nil {
```

Comment:
This seems like a bug. Worth logging at least? Or should we use ! here and fail if this happens?